src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+      data-qa="big"
+    >
+      <div class="input-wrapper">
+        <div class="icon-base icon-lg"></div>
+        <input
+          type="text"
+          name="mySearch"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          class="input-base input-lg"
```

Comment:
you can use big input style by default for inputs and change style for other inputs (try use BEM)

Look 2 point on [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)